kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java

```diff
@@ -442,9 +444,28 @@ protected Bitmap processBitmap(Object data) {
         kmAudioRecordManager = new KmAudioRecordManager(getActivity());
     }
 
+    private void setupChatBackground(){
+
+        if (!TextUtils.isEmpty(alCustomizationSettings.getChatBackgroundColorOrDrawable())){
+            String customChatBackground = alCustomizationSettings.getChatBackgroundColorOrDrawable();
+
+            if (customChatBackground.contains("#") && customChatBackground.length() >= 7){
+                conversationRootLayout.setBackgroundColor(Color.parseColor(alCustomizationSettings.getChatBackgroundColorOrDrawable()));
+            }else{
+                Resources resources = getResources();
+                int resourceId = resources.getIdentifier(customChatBackground, "drawable",getContext().getPackageName());
+                if (resourceId != 0){
+                    conversationRootLayout.setBackgroundResource(resourceId);
+                }
+            }
```

Comment:
If someone adds an invalid drawable or an invalid color, it should give an error message and set the default background color.

Reply:
@amntoppo The Above changes will set the default background only(If invalid input comes). But Do we really need to show the error in Toast?

Reply:
If default color will be set if invalid resource is coming, then it should be fine.